packages/bun-types/bun.d.ts

```diff
@@ -750,7 +750,12 @@ declare module "bun" {
       paths?: Record<string, string[]>;
       baseUrl?: string;
       /** "preserve" is not supported yet */
-      jsx?: "preserve" | "react" | "react-jsx" | "react-jsxdev" | "react-native";
+      jsx?:
```

Comment:
maybe lets not put `react-native` in there just to not give people to the wrong impression